cypress/support/step_defination/createDictionary/createDictionary.ts

```diff
@@ -0,0 +1,153 @@
+/// <reference types="cypress" />
+/// <reference types="../../" />
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+
+Given('the user is logged in', () => {
+  cy.login();
+});
+
+Given(' the user is on the Dictionaries page', () => {
```

Comment:
Leading spaces are significant, so watch out for those

```suggestion
Given('the user is on the Dictionaries page', () => {
```